### PR TITLE
New icons

### PR DESCRIPTION
**Pull request by agderes**
added new icons in the left menu

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "vue-flag-icon": "1.0.6",
     "vue-i18n": "8.16.0",
     "vue-router": "3.1.5",
-    "vuetify": "2.2.12",
+    "vuetify": "2.2.21",
```
Comment (1kh4l) [inline package.json:20]: is this change part of your task?

File: src/assets/sidenav-menu.json [hunk 2]
```diff
@@ -61,6 +61,27 @@
...
+            "title": "sidenav.support-us.title",
+            "icon": "mdi-charity",
+            "items": [
+                {
+                    "title": "sidenav.support-us.items.support-stu",
+                    "icon": "mdi-human-male-female"
+                }
+            ]
         }
+
```
Comment (1kh4l) [inline src/assets/sidenav-menu.json:85]: this space is not necessary

File: yarn.lock [hunk 3]
```diff
@@ -10,37 +10,28 @@
     "@babel/highlight" "^7.8.3"
 
 "@babel/core@^7.0.0":
-  version "7.8.4"
```
Comment (1kh4l) [inline yarn.lock:13]: is this yarn.lock part of your PR as well? why?

**Comment by 1kh4l** [comment]
Solve the conflicts from the branch and if you haven't created any Vue components then don't add the empty files

File: src/locales/co.json [hunk 4]
```diff
@@ -30,7 +30,19 @@
                 "courses"   : "Electivas",
                 "study-plan": "Currículos"
             }
-        }
+        },
+        "insa-access": {
+            "title": "Acceso al INSA",
+            "items": {
+                "admissions-processes": "Proceso de admisión"
```
Comment (1kh4l) [inline src/locales/co.json:37]: Admisiones can be

File: src/locales/co.json [hunk 5]
```diff
@@ -30,7 +30,19 @@
                 "courses"   : "Electivas",
                 "study-plan": "Currículos"
             }
-        }
+        },
+        "insa-access": {
+            "title": "Acceso al INSA",
```
Comment (1kh4l) [inline src/locales/co.json:35]: Can you do some research in universities to see how they handle sections like this ?

Comment (agderes) [reply src/locales/co.json:35]: I did and I saw that section on the oxford website :)

**Pull request merged**